packages/utils/parcel-watcher-watchman-js/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@parcel/watcher-watchman-js",
+  "version": "2.12.0",
```

Comment:
Maybe confirm with @devongovett what version new packages should be on?

Reply:
Confirmed this is what the version should be set to